benches/bench.rs

```diff
@@ -58,6 +61,27 @@ fn grow_by_insertion(b: &mut Bencher) {
     });
 }
 
+#[bench]
+fn test_insertion_removal(b: &mut Bencher) {
+    let mut m: HashSet<Vec<u8>> = HashSet::new();
+    let tx: Vec<Vec<u8>> = (0..4096)
+             .map(|_| 
+                (rand::thread_rng().gen_iter::<u8>().take(16)).collect())
+             .collect();
+
+    b.iter(|| {
+        for i in 0..4096 {
+            assert_eq!(m.contains(&tx[i].clone()), false);
+            assert_eq!(m.insert(tx[i].clone()), true);
+        }
+        for i in 0..4096 {
+            println!("removing {} {:?}", i, tx[i]);
+            assert_eq!(m.remove(&tx[i]), true);
+        }
+    });
+}
+
+
```

Comment:
Delete this since we have the version in tests/set.rs now.

Reply:
This is not resolved. You should revert your changes to the benchmarks file and only keep the file in `tests`.